Author: Jordi Mallach <jordi@debian.org>
Description: Fix build errors caused by -Werror=format-security.
 When compiled with hardening-flags enabled, ld10k1 fails to build due to
 a pair of unsafe printf usages that trigger -Werror=format-security.
Forwarded: no

Index: alsa-tools-1.0.25/ld10k1/src/lo10k1.c
===================================================================
--- alsa-tools-1.0.25.orig/ld10k1/src/lo10k1.c	2012-05-19 17:02:04.000000000 +0200
+++ alsa-tools-1.0.25/ld10k1/src/lo10k1.c	2012-05-19 17:08:40.460390666 +0200
@@ -868,7 +868,7 @@
 
 void debug_print(char *str)
 {
-	printf(str);
+	printf("%s", str);
 }
 
 static int debug(int deb)
Index: alsa-tools-1.0.25/ld10k1/src/ld10k1.c
===================================================================
--- alsa-tools-1.0.25.orig/ld10k1/src/ld10k1.c	2012-05-19 17:16:35.296366397 +0200
+++ alsa-tools-1.0.25/ld10k1/src/ld10k1.c	2012-05-19 17:17:21.356364047 +0200
@@ -62,7 +62,7 @@
 			 localtime(&tp));
 		fprintf(out, "%s %s", timestr, label);
 	} else 
-		fprintf(out, label);
+		fprintf(out, "%s", label);
 	vfprintf(out, fmt, va);
 	fprintf(out, "\n");
 	fflush(out);
